docs/api/cases/cases-api-create.asciidoc

```diff
@@ -0,0 +1,203 @@
+[[cases-api-create]]
+== Create case API
+++++
+<titleabbrev>Create case</titleabbrev>
+++++
+
+Creates a case.
+
+=== Request
+
+`POST <kibana host>:<port>/api/cases`
+
+=== Prerequisite
+
+You must have `all` privileges for the *Cases* feature in the *Management*,
```

Comment:
> You must have `all` privileges for the *Cases* feature in the *Management*,

I had thought I would also need privileges for the *Actions and connectors* feature, but that didn't seem to be the case in my simple tests.

Reply:
You don't need privileges for the Actions and connectors feature to create a case. But you cannot push a case if you don't have.